internal/tiltfile/extension/extension.go

```diff
@@ -46,6 +47,12 @@ func (e *Extension) LocalPath(t *starlark.Thread, arg string) (string, error) {
 		return "", nil
 	}
 
+	loadIsHappeningInTopLevel := t.CallStackDepth() == 1
+
+	if !loadIsHappeningInTopLevel {
+		return "", fmt.Errorf("Illegal extension load: extensions must be loaded from the root Tiltfile. This extension was loaded from %s", t.CallFrame(t.CallStackDepth()-1).Pos.Filename())
```

Comment:
confused about why your error message needs to manually include the filename. shouldn't skylark already print the full stack trace here? do you understand why it doesn't?

Reply:
Ah it looks like it does, I just wasn't looking far enough to the left.